BlueprintUI/Tests/BlueprintViewTests.swift

```diff
@@ -145,7 +145,10 @@ class BlueprintViewTests: XCTestCase {
             ]
         )
 
+        blueprintView.layoutIfNeeded()
+
         tags = [1, 2]
+
         for index in blueprintView.currentNativeViewControllers.indices {
```

Comment:
Calling `currentNativeViewControllers` implicitly lays out the hierarchy, which is a bit non-obvious; so I added the explicit `layoutIfNeeded` calls.